src/qt/overviewpage.cpp

```diff
@@ -152,6 +154,23 @@ void OverviewPage::handleOutOfSyncWarningClicks()
     Q_EMIT outOfSyncWarningClicked();
 }
 
+void OverviewPage::setPrivacy(bool privacy)
+{
+    m_privacy = privacy;
+    if (walletModel && walletModel->getOptionsModel()) {
+        if (m_balances.balance != -1) {
+            setBalance(m_balances);
+        }
+
+        ui->listTransactions->setVisible(!m_privacy);
+    }
+
+    const QString status_tip = m_privacy ? tr("PRIVACY mode is activated. To reveal cloaked values uncheck Menu->Settings->Mask values") : "";
```

Comment:
suggestion:
```diff
- "PRIVACY mode is activated. To reveal cloaked values uncheck Menu->Settings->Mask values"
+ "PRIVACY mode is activated. To unmask the values, uncheck Menu->Settings->Mask values."
```
- adds a missing "the", comma, and period
- replaces "reveal cloaked" with the more consistent, shorter, and simpler "unmask"

(also, maybe consider not screaming :) e.g. s/PRIVACY/Privacy/)

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16432#issuecomment-633063508).